### PR TITLE
Do not install Objective-C bridge header for Swift.

### PR DESCRIPTION
**Pull request by wuhao5**
The EarlGrey.swift is only a wrapper for ease of use in Swift, it causes some issue here #636 where it complains about the EarlGrey macro and EarlGrey class when importing the bridge header.

### CODE REVIEW

File: EarlGrey.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1286,6 +1286,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = com.google.earlgrey.EarlGrey;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SKIP_INSTALL = NO;
+				SWIFT_INSTALL_OBJC_HEADER = NO;
```
Comment (tirodkar) [inline EarlGrey.xcodeproj/project.pbxproj]: To confirm, this basically stops the Objc Compatibility Header from being created, right? Any reason why we would want that?

Comment (wuhao5) [reply EarlGrey.xcodeproj/project.pbxproj]: mentioned here: https://github.com/google/EarlGrey/issues/636#issuecomment-334890424

**Comment by khandpur** [comment]
ping.

**Comment by tirodkar** [comment]
Waiting on the failing Ruby job to run again.

**Comment by tirodkar** [comment]
This is probably unrelated, but do you know why the Ruby Test is failing here?

**Comment by wuhao5** [comment]
Attempted to fix the warning but it turns out to be a false positive, disabling this check.

**Comment by khandpur** [comment]
Could we send a different PR unless the two commits are closely related?

**Comment by bootstraponline** [comment]
> Could we send a different PR unless the two commits are closely related?

I think it's only four lines of code so there's no issue. The PR can be merged via rebasing so that the individual commits are preserved to represent the logical grouping of work.

**Comment by khandpur** [comment]
Sounds good. @tirodkar feel free to merge if it's okay with you.

**Comment by bootstraponline** [comment]
earlgrey_xcode_9_swift

When I try this branch in our project, I get no such module.

> pod 'EarlGrey', git: 'https://github.com/wuhao5/EarlGrey.git', branch: 'swift-bridge-off'

It works using the released version. Am I doing something wrong?

**Comment by bootstraponline** [comment]
After talking with our iOS dev lead, I think `SWIFT_INSTALL_OBJC_HEADER = YES` is what allows Swift to import OBJC dependencies.

**Comment by bootstraponline** [comment]
`SWIFT_INSTALL_OBJC_HEADER = NO` on my `SoGrey` framework which imports `EarlGrey` resolved the issue. Thanks @wuhao5 for solving this.

**Comment by wuhao5** [comment]
Changes made to EarlGrey will not actually fix it. Closing this.

**Pull request closed without merging**